src/pages-conditional/eth.tsx

```diff
@@ -16,153 +25,233 @@ import InfoBanner from "../components/InfoBanner"
-  background-size: cover;
-  background-repeat: no-repeat;
-  margin-left: 2rem;
-  @media (max-width: ${(props) => props.theme.breakpoints.m}) {
-    margin-left: 0;
-    width: 100%;
-  }
-`
+const Slogan = (props: ChildOnlyProp) => (
+  <Text
+    textStyle="normal"
+    fontWeight="normal"
+    fontSize="2rem"
+    lineHeight={1.4}
+    {...props}
+  />
+)
 
-const Header = styled.header`
-  flex: 1 1 50%;
-  min-width: 300px;
-  margin-top: 8rem;
-  @media (max-width: ${(props) => props.theme.breakpoints.m}) {
-    margin-top: 1.5rem;
-  }
-`
+const Title = (props: ChildOnlyProp) => (
+  <Heading
+    as="h1"
+    fontSize="sm"
+    lineHeight={1.4}
+    letterSpacing="0.04em"
+    fontWeight="500"
+    mb={4}
+    mt={0}
+    textTransform="uppercase"
+    color="textTableOfContents"
+    {...props}
+  />
+)
 
-const H2 = styled.h2`
-  margin-top: 0;
-`
+const Subtitle = (props: ChildOnlyProp) => (
+  <Text fontSize="xl" lineHeight={1.4} color="text200" {...props} />
+)
 
-const H4 = styled.h4`
-  font-weight: 600;
-`
+const TwoColumnContent = (props: FlexProps) => (
+  <Flex
+    w="full"
+    direction={{ base: "column", lg: "row" }}
+    justify="space-between"
+    p={8}
+    mb={12}
+    {...props}
+  />
+)
 
-const StyledCardContainer = styled(CardContainer)`
-  margin-bottom: 2rem;
-`
+export const LeftColumn = (props: ChildOnlyProp) => (
+  <Box
+    flex="0 0 50%"
+    maxW={{ base: "full", lg: "75%" }}
+    mr={{ lg: 16 }}
+    {...props}
+  />
+)
 
-const TokenCard = styled(HorizontalCard)`
-  min-width: 100%;
-  margin: 0.5rem 0rem;
-  border-radius: 0px;
-`
+export const RightColumn = (props: ChildOnlyProp) => (
+  <Flex
+    flex="0 1 50%"
+    direction="column"
+    justify="center"
+    maxW={{ base: "full", lg: "75%" }}
+    mt={{ base: 12, lg: 0 }}
+    {...props}
+  />
+)
 
-const TextDivider = styled.div`
-  margin-bottom: 2rem;
-  margin-top: 2rem;
-  width: 10%;
-  height: 1px;
-  background-color: ${(props) => props.theme.colors.searchResultBackground};
-  @media (min-width: ${(props) => props.theme.breakpoints.l}) {
-    align-self: flex-start;
-  }
-`
+const SubtitleTwo = (props: ChildOnlyProp) => (
+  <Text fontSize="xl" lineHeight={1.4} color="text300" mb={8} {...props} />
+)
 
-const CentralColumn = styled.div`
-  display: flex;
-  flex-direction: column;
-  align-items: center;
-  max-width: 960px;
-  margin: 4rem auto;
-`
+const HeroContainer = (props: ChildOnlyProp) => (
+  <Flex
+    align="center"
+    justify="space-between"
+    direction={{ base: "column-reverse", md: "row" }}
+    {...props}
+  />
+)
 
-const CentralActionCard = styled(ActionCard)`
-  flex: none;
-  margin: 2rem 0;
-  .action-card-image-wrapper {
-    padding: 1rem;
-  }
-  @media (min-width: ${(props) => props.theme.breakpoints.s}) {
-    display: flex;
+const Hero = (props: ChildOnlyProp) => (
+  <Box
+    flex="1 1 100%"
+    maxW="800px"
+    bgSize="cover"
+    bgRepeat="no-repeat"
+    ml={{ base: 0, md: 8 }}
```

Comment:
`ms` for `margin-inline-start` (rtl responsive)
```suggestion
    ms={{ base: 0, md: 8 }}
```